### PR TITLE
func/gwe-51-checkout

### PR DESCRIPTION
**Pull request by joshxfi**
*recycled zomer's code for showing cart details on frontend

data sent:
```ts
type IOrderStatus =
  | "TO_PAY"
  | "TO_SHIP"
  | "TO_RECEIVE"
  | "DELIVERED"
  | "CANCELLED";

// temporary
type IOrderMethod = "CREDIT_CARD" | "DEBIT_CARD" | "PAYMAYA" | "GCASH" | "COD";

export interface IOrderItem {
  buyerId: string;
  shopId: string;
  products: string[];
  status: IOrderStatus;
  orderDate: any;
  method: IOrderMethod;
}
```
method sent is temporarily `"COD"` till there is other payment methods.

### CODE REVIEW

**Comment by joshxfi** [comment]
forgot to remove context

File: src/models/user/index.d.ts [hunk 1]
```diff
@@ -48,3 +48,22 @@ export interface IUserCartItem {
   quantity: number;
   selected: boolean;
 }
+
```
Comment (princejoogie) [inline src/models/user/index.d.ts:51]: move these types in models/orders

**Review by princejoogie — CHANGES_REQUESTED** [review]

**Comment by princejoogie** [comment]
@zomeru

@joshxfi made a cart context here so might be usable in your next change. cos we should only have 1 cart context

**Comment by joshxfi** [comment]
cleaned up context so it can be used, & also moved Cart component to the page itself since it was the only component there.

**Pull request merged**